content/reference/endpoints/payment_gateways.md

```diff
@@ -50,3 +76,34 @@ GET /payment_gateways/:payment_gateway_id
 <%= render 'json_response', endpoint: "payment_gateways", scopes: %w(preferences-payments-read) %>
 
 
+## Create a new Payment Gateway
+
+Creates a Payment Gateway
+
+~~~
+POST /payment_gateways
+~~~
+
+<%= render 'json_response', endpoint: "payment_gateways", request: "create",
+scopes: [{ preferences_payments_write: "preferences-payments-read" }] %>
+
+## Update a payments
+
+Returns an updated living_room identified by ID.
+
+~~~
+PUT /payment_gateways/:payment_gateway_id
+~~~
+
+<%= render 'json_response', endpoint: "payment_gateways", request: "update",
+scopes: [{ preferences_payments_write: "preferences-payments-read" }] %>
+
+## Destroy a living_room
```

Comment:
I guess it was about payment gateway 😄 Can you please remove this action? Soon it won't be possible to submit such request until proper handling for removal of gateways is introduced in our system.

Reply:
@adamgrad We cannot remove this endpoint as it would be a breaking change. Why not just change copy & paste stuff and update it in the code? The endpoint will still be there, it will just be a soft-delete instead of hard-delete

Reply:
@Azdaroth first of all, it wasn't a documented endpoint so even though it existed it was not recommended anywhere to use it.
Secondly, even if we soft delete the gateway, api v3 still won't return it (as our api returns only non-canceled records) and we will have failures while processing any payment that was scheduled before someone decided to remove the gateway. Currently product does not define any flow for either us or CC team to handle upcoming payments for the gateway that someone wants to drop, hence even soft deletion is going to create mess and, lot of support and potential friction with channels, just like we had for the last user who deleted his gateway with multiple upcoming payments.

Reply:
Ok, let's remove it then